triggers/daml/BUILD.bazel

```diff
@@ -39,21 +39,23 @@ dependencies:
   - daml-script.dar
 build-options: {build_options}
 EOF
-          $(location //compiler/damlc) build --project-root=$$TMP_DIR -o $$PWD/$@
+          $(location //compiler/damlc) build --project-root $$TMP_DIR \
+              --cpp $$PWD/$(location @stackage-exe//hpp) \
+              -o $$PWD/$@
           rm -rf $$TMP_DIR
         """.format(
             build_options = str([
-                "--ghc-option",
-                "-Werror",
```

Comment:
Without which

```
hpp: Couldn't open input file: -Werror
CallStack (from HasCallStack):
  error, called at src/Hpp/CmdLine.hs:103:22 in hpp-0.6.1:Hpp.CmdLine
```

Reply:
Probably worth opening an issue for. Ideally, we would fix whatever is broken here and get back to the point where we can enable it.